toolbox/nfd/deploy_from_commit.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source ${THIS_DIR}/../_common.sh
+
+if [ "$#" -lt 2 ]; then
```

Comment:
@kpouget the ad-hoc / messy arg parsing each script does now is getting a bit out of hand, maybe we should consider transforming the toolbox from a bunch of shell scripts to something like https://click.palletsprojects.com/en/8.0.x/ or https://github.com/google/python-fire

This will be more maintainable / give us `--help` for free

Reply:
fair point @omertuc, totally makes sense.
Another possibly backward incompatible change on the way, most likely

Reply:
pls not python, `ci-artifacts` should be friendly to all red hat engineers at least, meaning, Ansible + bash as long as possible (I get python for Ansible plugins, but not for main functionality)

Reply:
@ArangoGutierrez I can't agree, the `toolbox` scripts are minimal entrypoints to Ansible, and most of them is copy-and-paste of any random other, no need to know (barely to understand) the language to create a new script!
on the other hand, a reliable parameter declaration/parsing would be very valuable to people to who wants to use the toolbox.

I'm not sure if we could find something suitable for `bash`, I remember some "tools" using the `usage` text as the way to define the command-line arguments, but not sure what it was ...

Reply:
thanks for the help message :)